src/rust/rust-proto-new/src/graplinc/grapl/api/plugin_registry/v1beta1.rs

```diff
@@ -1 +1,495 @@
+pub use crate::protobufs::graplinc::grapl::api::plugin_registry::v1beta1::{
```

Comment:
This breaks encapsulation, which we should not do. The point of `rust-proto-new` is to encapsulate all the yucky protobuf compiler artifacts and expose instead our own SerDe types.

Reply:
I'm a tad confused, this is already in rust-proto-new